nemo/core/classes/common.py

```diff
@@ -81,16 +82,21 @@ def _attach_and_validate_output_types(self, out_objects):
                     res.neural_type = out_types_list[ind][1]
 
 
-class Serialization(ABC):
+class Serialization:
     @classmethod
     def from_config_dict(cls, config: DictConfig):
         """Instantiates object using DictConfig-based configuration"""
+
         if 'cls' in config and 'params' in config:
-            # regular hydra-based instantiation
-            instance = hydra.utils.instantiate(config=config)
+            class_obj = eval(config.cls)
```

Comment:
Raw eval to inflate class ?? This is generally considered highly dangerous behavior throughout python. You can inject any class string here.

Reply:
@titu1994 you are right, I know, this is far from perfect, just wanted to a) have "save/restore" working and b) unblock our CI ASAP - and I spent more than a day working on this already.

I can work on this as part of Serialization PR - I will have to change plenty of things in here to make it work with graphs anyway...

Reply:
Tom, there was nothing blocking CI - all tests were running, albeit using unittests rather than pytests. Oleksii said he would fix serialization, but you have gone ahead and merged this solution. At this point I dont understand how this PR got approved with this kind of class instantiation, and I just hope future PR fixes this as a high priority patch.

Reply:
"Oleksii Kuchaiev  16 minutes ago: 
let’s merge - we have 2 approvals now. I’ll have a look at WER test and try to debug it"